.gitignore

```diff
@@ -2,4 +2,3 @@
 [._]*.sw[a-p]
 Cargo.lock
 target
-tests/
```

Comment:
is there a reason why `tests` directory was ignored?

Reply:
🤷🏻‍♂️ ... I guess that @burrbull was generating some files automatically and did not want to version them in the past? Can't see any reason to keep ignoring it TBH.

Reply:
It is not my repo initially :)